src/app/components/confirm/index.html

```diff
@@ -137,7 +137,7 @@
         </li>
 			</ul>
 			<p>
-        The answer portion of the dialog can contain up to 3 choices that are represented as terminal actions (buttons).
+        The answer portion of the dialog can contain up to 3 choices that are represented as terminal actions (buttons) and should reiterate the action being taken. For example, if the question is, "Are you sure you want to delete this draft?", the primary button text should display "Delete".
```

Comment:
In the first sentence, what do you think of changing "and" to "that"?

Also, just a thought, but the punctuation in the second sentence is distracting. What do you think of: For example, use "Delete" for the primary button text if the question is "Are you sure you want to delete this draft?"

Reply:
Actually, looking back at the UA Patterns content, I have a different suggestion for the first sentence. I'd flip the current order: "The answer portion of the dialog answers the question or tells users what will happen. It can contain up to 3 choices represented as terminal actions (buttons) that should reiterate the action to take."